### ISSUE TITLE
[BUG][JavaSpring] performBeanValidation=true ambiguous import for @Email

### ISSUE THREAD

**Issue by nickshoe** [OpenAPITools/openapi-generator#5637]
Using openapi-generator-maven-plugin v4.2.2.

I had to enable `performBeanValidation` configuration, since the `format: email` validation through the `@Email` annotation seems to be only rendered by the mustache template this way (`beanValidationCore.mustache` line 13-15).

The following schema specification:

```yaml
    AccountUpdateCommand:
      type: object
      required:
        - email
      properties:
        email: 
          type: string
          format: email
```

leads to:

```java
  /**
   * Get email
   * @return email
  */
  @ApiModelProperty(required = true, value = "")
  @NotNull

@Email
  public String getEmail() {
    return email;
  }
```

but the @Email annotation exists in more than one of the imported packages:

```java
...
import javax.validation.constraints.*;
import org.hibernate.validator.constraints.*;
...
```

So the IDE gives the following error:
"The type Email is ambiguous"

Furthermore, the `org.hibernate.validator.constraints.Email` annotation is **deprecated** in favor of the standard `javax.validation.constraints.Email`.

Questions/Considerations:

- What's the real purpouse of `performBeanValidation` configuration in contrast to `useBeanValidation` (deafult) configuration? 
  - It looks to me that it only affects the package used for annotations related to validation. 
  - Do we really need it? Or could we unify this parameter with `useBeanValidation`, and let it specify the library to use for validation?
- If `performBeanValidation=false`, the `@Email` annotation should be rendered anyway, since the `javax.validation.constraints` package has it.

**Comment by nickshoe** [comment]
Duplicate of #4786

**Comment by nickshoe** [comment]
I hadn't considered the Java version in my considerations above.

**Comment by nickshoe** [comment]
Closing this, since it has been solved by #4788.
